prac_09/band.py

```diff
@@ -0,0 +1,16 @@
+""""""
+
+
+class Band():
+    def __init__(self, name=""):
+        self.name = name
+        self.members = []
+
+    def __str__(self):
+        return f"{self.name} ({','.join([str(member) for member in self.members])})"
+
+    def add(self, member):
```

Comment:
In your method name 'add' it would be better if you consider using 'add_member' because of good naming 🤔